src/components/BadTextInput.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <v-text-field
+    @input="update"
+    :disabled="disabled"
+    :label="label"
+    :placeholder="placeholder"
+    :id="id"
+    :value="value"
+  >
+  </v-text-field>
+</template>
+<script>
+export default {
+  name: "BadTextInput",
+  data() {
+    return {};
+  },
+  methods: {
+    update(newValue) {
```

Comment:
Maybe I'm being react-ey but I'd like it if this method name started with `handle-` or `on-`:

```
handleInput(newValue) {
    
}
```

Reply:
updated